util/src/main/java/tech/pegasys/artemis/util/cli/CommandLineArguments.java

```diff
@@ -48,13 +37,6 @@
       description = "Path/filename of the config file")
   private String configFile = "./config/config.toml";
 
-  @Option(
-      names = {"-s", "--sim"},
-      arity = "0",
-      paramLabel = "<FILENAME>",
-      description = "PoW simulation flag, w/ optional input file")
-  private String inputFile = null;
-
   @Option(
       names = {"-f", "--format"},
```

Comment:
go ahead and remove this flag too and put it in the toml config under output.

Reply:
Now i have moved "format" flag to config. Pls review.